code/modules/projectiles/guns/energy.dm

```diff
@@ -477,7 +479,7 @@
 	accuracy_mult_unwielded = 0.5 //Heavy and unwieldy; you don't one hand this.
 	scatter_unwielded = 100 //Heavy and unwieldy; you don't one hand this.
 	damage_falloff_mult = 0.25
-	fire_delay = 3
```

Comment:
uhhhhh???

Reply:
standard lasgun mode has a fire_delay of 2. And we start in this mode

Reply:
ok